proprietary/assets/README.md

```diff
@@ -3,3 +3,7 @@
 # Assets
 
 Assets for the City of Amsterdam
+
+## Web app manifest, app icons and favicon
+
+See the Design System documentation for an [overview and examples of the manifest and these icons](https://amsterdam.github.io/design-system/?path=/docs/docs-assets-favicon-app-icons--docs).
```

Comment:
This URL is about to change to designsystem.amsterdam.

Reply:
Check, I'll adjust it then.

Reply:
Aram and I will be looking into this on Friday. We have more links to this domain, we could also leave this one like this and update all of them when we’re actually migrating.

Reply:
Have the links already changed and can I update them? Or should I leave it as is?

Reply:
No, not yet. The domain name is ready; next step is installing an SSL certificate.